package com.sds.exception;

public class DBTest{

	public DBTest() {
	
	
	
	}
	
	public void test() throws ArrayIndexOutOfBoundsException{
		
		int[] arr = new int[2];
		arr[0] = 1;
		arr[1] = 2;
		
		System.out.println(arr[2]);
		
		/*try {
			
			System.out.println(arr[2]);
		
		} catch (Exception e) {
			
			//e.printStackTrace();
			System.out.println("배열의 크기를 넘었습니다.");
		
		}*/
		
	}
	
	public static void main(String[] args) {
		
		DBTest dbTest =  new DBTest();
		
		try {
			
			dbTest.test();
		
		} catch (ArrayIndexOutOfBoundsException e) {
			
			e.printStackTrace();
		}
		
	}
	
}
